content/lessons/genesis/messages.js

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl'
```

Comment:
I think it makes sense to rename this file from `messages` to `translations`.

Reply:
All the react-intl docs, tooling, and examples all refer to them as `messages` though. Are we sure we want to use `translations` rather than `messages`?

Reply:
Also, these files do not include any translations. The translations are what ends up in the locale JSON files.

Reply:
Yes, I think for people who are not familiar with `react-intl` the word `translations` makes more sense than `messages` but this might just be my personal preference. I feel it communicates the intent more clearly. If users need to visit react-intl docs I think our translation system is too complicated anyway, but curious to learn what others think about this.

Reply:
react-intl might be the wrong choice then. One of the main benefits of using react-intl is to use all the components they provide. You'll have to read the docs in order to take advantage of all the benefits their components provide.